recipes/pyro5/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "5.5.1" %}
+
+package:
+  name: pyro5
+  version: {{ version }}
+
+source:
+  fn: Pyro5-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/P/Pyro5/Pyro5-{{ version }}.tar.gz
+  sha256: fe7998b0bfab7e6f1bf02fa7d69e4b4c53305da745bae07d604eeea139316828
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
I think this recipe can be ``noarch: python``. You can select the ``python`` version on ``host`` and ``run`` requirements

Reply:
Hm, I started with that, and then it went on to build it with python 2.7 which failed. Pyro5 is for python 3.5+   Is there something else I should have set perhaps?

Reply:
See the other suggestions, they explain it.

Reply:
ok I will try!